stl/inc/ostream

```diff
@@ -912,15 +912,46 @@ basic_ostream<char, _Traits>& operator<<(
 }
 
 #ifdef __cpp_char8_t // These deleted overloads are specified in P1423.

```

Comment:
Note to other reviewers: This means the `char8_t` overloads continue to be deleted even in sub-C++20 mode (e.g., if compiling with e.g. `/std:c++17 /Zc:char8_t`). I'm fine with this inconsistent behavior of `char8_t` vs. `char16_t`/`char32_t`, shout if you are not.

Reply:
That's fine, in any case only "he who likes pain" will use `char8_t` in C++20... Now, where is that Jens Gustedt pdf on modern C ... sorry talking to myself, have to go now... Good work Casey, thanks a lot, see you ...